tools/test

```diff
@@ -147,6 +135,8 @@ fi
 
 if [ -n "$SLOW" ] && [ -z "$COVERDIR" ]; then
     go get github.com/weaveworks/tools/cover
+    go mod vendor # re-vendor everything
```

Comment:
What about putting the `cover` tool into the build image, like we do with various other tools?
https://github.com/cortexproject/cortex/blob/b64dabe749023da882c9739b216de683194b9bfb/build-image/Dockerfile#L19-L23

Reply:
I will send PR to include cover tool in build image.